roles/cassandra_install/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 # defaults file for cassandra_install
 cassandra_packages: cassandra
+cassandra_exact_version: 3.11.17
```

Comment:
The default for this should probably be unset so the latest package is installed unless otherwise specified.